torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -586,6 +568,9 @@ def view_avoid_dupes_with_primals(t):
         # inference and training paths decide which inputs/output to ask for subclass info on.
         # However, we currently stash indexing information on each SubclassMeta about its order
         # in the graph outputs list.
+
+        # TODO (dberard) make this use the same logic as schemas.py??
+        # TODO (dberard) why does it think that nothing requires grad after desugaring TwoTensor subclass
```

Comment:
Your PR looks strictly better (happy to approve) - although agreed that this would be a nice refactor too

Reply:
this comment was left over from some debugging, thanks for catching it (and for the review!)